### PR TITLE
FP-14 React Router

### PR DESCRIPTION
**Pull request by thisisofficial**
### Description

As per said by Carlos, i added some mock routes to start up the Router in our app.

### Testing Checklist

- [] Test cases have been added/updated
- [x] Tested locally in browser (at least one other than Chrome)
- [x] All tests pass

### What JIRA Ticket(s) does this PR address?

- [FP-14](https://cd95.atlassian.net/browse/FP-14)

### CODE REVIEW

File: src/pages/DataGrid.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { columns } from '../utils/tableColumns';
+import { useApi } from '../hooks/useApi';
+import { Table } from '../components/table/Table';
+
+export function DataGrid() {
+  const { fetchData } = useApi();
+  const rows: GridRowsProp = fetchData;
+
+  return (
+    <div>
```
Comment (caarlosdamian) [inline src/pages/DataGrid.tsx]: no necesitas este div usa <></>

File: src/App.tsx [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
 
 function App() {
-  const { fetchData } = useApi();
-
-  const rows: GridRowsProp = fetchData;
-
   return (
     <div className="App">
-      <Table columns={columns} rows={rows} />
+      <BrowserRouter>
```
Comment (caarlosdamian) [inline src/App.tsx]: Esto va en /routes  no debe ir en App

**Review by caarlosdamian — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 2 above)
Comment (thisisofficial) [reply src/App.tsx]: Fixed.

File: src/pages/DataGrid.tsx (hunk 1 above)
Comment (thisisofficial) [reply src/pages/DataGrid.tsx]: Removed.

File: src/pages/DataGrid.tsx [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { GridRowsProp } from '@mui/x-data-grid';
```
Comment (caarlosdamian) [inline src/pages/DataGrid.tsx]: Este componente va en /components no en /pages

File: src/pages/index.tsx [hunk 4]
```diff
@@ -0,0 +1,2 @@
+export { DataGrid } from './DataGrid';
```
Comment (caarlosdamian) [inline src/pages/index.tsx]: Este va en components

**Review by caarlosdamian — DISMISSED** [review]

File: src/pages/DataGrid.tsx (hunk 3 above)
Comment (thisisofficial) [reply src/pages/DataGrid.tsx]: Is this what you meant?

**Review by caarlosdamian — APPROVED** [review]

**Pull request merged**